conformance/tests/httproute-cross-namespace.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+metadata:
+  name: httproute-cross-namespace
```

```suggestion
  name: web-backend
```

We should avoid putting namespace/name/types in the Name or Namespace keys. That makes the whole thing much harder to parse and often unnecessarily.

---

I definitely agree with the idea of removing type prefixes like `httproute-`, I've done that. I'm less sold on giving the routes more generic names. I think names can be used to provide context on what a Gateway/HTTPRoute is for. For example, a Gateway with a name `same-namespace-only` could make it easier to read and understand a test than a Gateway with a more generic name like `app-lb`.